.github/workflows/node.js.yml

```diff
@@ -60,8 +59,8 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
           registry-url: https://npm.pkg.github.com/
-          scope: "@manekinekko"
-      - run: echo "registry=https://npm.pkg.github.com/@manekinekko" >> .npmrc
+          scope: "@azure"
+      - run: echo "registry=https://npm.pkg.github.com/@azure" >> .npmrc
       - run: npm publish $(ls *.tgz)
         env:
-          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
\ No newline at end of file
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
We probably need to regenerate a new token. I'll check if I have permissions.

Reply:
Not needed, that token is provided by GitHub Actions automagically

Reply:
😃